com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -38,231 +30,317 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem
 
 #pragma warning restore IDE1006 // restore naming rule violation check
 
-#if DEVELOPMENT_BUILD || UNITY_EDITOR
-        private static ProfilerMarker s_SyncTime = new ProfilerMarker($"{nameof(NetworkManager)}.SyncTime");
-        private static ProfilerMarker s_TransportPoll = new ProfilerMarker($"{nameof(NetworkManager)}.TransportPoll");
-        private static ProfilerMarker s_TransportConnect = new ProfilerMarker($"{nameof(NetworkManager)}.TransportConnect");
-        private static ProfilerMarker s_HandleIncomingData = new ProfilerMarker($"{nameof(NetworkManager)}.{nameof(HandleIncomingData)}");
-        private static ProfilerMarker s_TransportDisconnect = new ProfilerMarker($"{nameof(NetworkManager)}.TransportDisconnect");
-#endif
-
-        private const double k_TimeSyncFrequency = 1.0d; // sync every second
-        private const float k_DefaultBufferSizeSec = 0.05f; // todo talk with UX/Product, find good default value for this
-
-        internal static string PrefabDebugHelper(NetworkPrefab networkPrefab)
+        // TODO 2023-Q2: Team discussion
+        // Removing this is a "breaking change"
+        // (the issue with interfaces...implementations have to be public!)
+        public void NetworkUpdate(NetworkUpdateStage updateStage)
         {
-            return $"{nameof(NetworkPrefab)} \"{networkPrefab.Prefab.name}\"";
-        }
-
-        internal NetworkBehaviourUpdater BehaviourUpdater { get; set; }
 
-        internal void MarkNetworkObjectDirty(NetworkObject networkObject)
-        {
-            BehaviourUpdater.AddForUpdate(networkObject);
         }
 
-        internal MessagingSystem MessagingSystem { get; private set; }
-
-        private NetworkPrefabHandler m_PrefabHandler;
+        /// <summary>
+        /// The client id used to represent the server
+        /// </summary>
+        public const ulong ServerClientId = 0;
 
-        internal Dictionary<ulong, ConnectionApprovalResponse> ClientsToApprove = new Dictionary<ulong, ConnectionApprovalResponse>();
+        /// <summary>
+        /// Gets the networkId of the server
+        /// </summary>
+        private ulong m_ServerTransportId => ConnectionManager.ServerTransportId;
```

Comment:
Style question... is `m_` still actually the appropriate prefix for a property? Feels a little odd to me.

Reply:
any private property requires the m_ prefix (as I understand it).